public/app/features/plugins/admin/helpers.ts

```diff
@@ -425,3 +425,33 @@ export function filterByKeyword(plugins: CatalogPlugin[], query: string) {
   }
   return idxs.map((id) => getId(dataArray[id]));
 }
+
+export function isPluginUpdateable(plugin: CatalogPlugin) {
+  // If there is no update available, the plugin cannot be updated
+  if (!plugin.hasUpdate) {
+    return false;
+  }
+
+  // Provisioned plugins cannot be updated
+  if (plugin.isProvisioned) {
+    return false;
+  }
+
+  // Core plugins cannot be updated
+  if (plugin.isCore) {
+    return false;
+  }
+
+  // Currently renderer plugins are not supported by the catalog due to complications related to installation / update / uninstall.
+  if (plugin.type === PluginType.renderer) {
+    return false;
+  }
+
+  // Preinstalled plugins (with specified version) cannot be updated
+  if (plugin.isPreinstalled.withVersion) {
+    return false;
+  }
+
+  // Managed plugins cannot be updated
+  return !plugin.isManaged;
```

Comment:
there is a state in cloud where we requested the plugin update, the plugin is updated in grafana.com, but it's not installed in locally yet. We are marking this with the `plugin.isUpdatingFromInstance` property. In this state the plugin can not be updated. This property is `undefined` in OSS and has value `true` when the version of a local plugin is different from the version of a instance plugin (taken from grafana.com).

We should include this here, otherwise the plugins that are is this state will keep being shown by the "New Updates" filter.

Reply:
Ah yes sorry I knew I forgot something when updating from your changes! Thanks for that